.travis.yml

```diff
@@ -42,7 +42,6 @@ matrix:
 
     allow_failures:
       - env: DJANGO=master
-      - env: DJANGO=2.1
```

Comment:
Drop this line for now and we’ll merge. We can add this after the next release. (Which should be the 16th.)

Reply:
Ok, removed ac396029.